asllib/AST.mli

```diff
@@ -272,7 +272,9 @@ type stmt_desc =
           represent the implicit throw, such as [throw;]. *)
   | S_Try of stmt * catcher list * stmt option
       (** The stmt option is the optional otherwise guard. *)
-  | S_Debug of expr
+  | S_Print of { args : expr list; debug : bool }
```

Comment:
Putting the constructor arguments in a record is really nice. That way, you can name them and document their intention.
I wish we had that for other constructors like E_Cond, for example.